alibi_detect/utils/frameworks.py

```diff
@@ -15,3 +18,90 @@
     has_sklearn = True
 except ImportError:
     has_sklearn = False
+
+
+HAS_BACKEND = {
+    'tensorflow': has_tensorflow,
+    'sklearn': has_sklearn,
+    'pytorch': has_pytorch,
+}
+
+
+def iter_to_str(iterable):
```

Comment:
Might be worth making this a private function if only intended to be used here. Otherwise needs a docstring.